.github/workflows/nightly.yml

```diff
@@ -0,0 +1,107 @@
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2020 NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Deploy a new version of the documentation or rebuild an existing one.
+
+# Note: All changes made to the gh-pages branch are non-destructive 
+#       (i.e. no force pushing) so all changes can be undone.
+
+name: nightly
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '35 0 * * *'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: configure python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+
+      - name: checkout cylc-doc
+        uses: actions/checkout@v2
+        with:
+          ref: master
+          path: docs
+
+      - name: install graphviz
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y graphviz pkg-config libgraphviz-dev
+          pip install pygraphviz
+
+      - name: install cylc-doc
+        run: |
+          pip install "${{ github.workspace }}/docs[all]"
+
+      - name: checkout cylc-flow
+        uses: actions/checkout@v2
+        with:
+          repository: cylc/cylc-flow
+          ref: master
+          path: cylc-flow
+
+      - name: install cylc-flow
+        run: |
+          # NOTE: Install with [all] so we can import plugins which may
+          #       have extra dependencies.
+          pip install './cylc-flow[all]'
+
+      - name: checkout gh-pages
+        uses: actions/checkout@v2
+        with:
+          ref: gh-pages
+          path: gh-pages
+
+      - name: install gh-pages
+        run: |
+          DOCS="${{ github.workspace }}/docs" \
+          PAGE="${{ github.workspace }}/gh-pages"  \
+
+          cd "$DOCS"
+          rm -r doc
+          ln -s "$PAGE" doc
+
+      - name: configure git
+        run: |
+          git config --global user.name "action:deploy"
+          git config --global user.email "action:deploy@github.com"
+
+      - name: remove previous nightly build
+        run: |
+          EDITOR='sed -i "s/pick \([0-9a-z]* -nightly build-\)/omit \1/"' \
```

Comment:
Use `sed` as the text editor in order to delete the offending commit via rebase.

Reply:
Shouldn't it be `drop` instead of `omit`?

Reply:
Yes, I think omit might have been a placeholder I failed to fix.